creator-cards/pierre-belin.json

```diff
@@ -2,28 +2,32 @@
   "name": "Pierre Belin",
   "sectionTitle": null,
   "countryCode": "fr",
-  "slogan": "",
+  "slogan": "Improve your code review, only for goats",
   "type": "CREATOR",
-  "bio": "",
-  "tags": ".NET, Akka.NET",
+  "bio": "C# & .NET Content Creator",
+  "tags": "Freelance Architecture / C#/.NET Developer",
   "socials": {
-    "youtube": "",
+    "youtube": "https://www.youtube.com/channel/UC5LwEdz8usxgvFPiFlelaQA",
     "linkedin": "https://www.linkedin.com/in/pierre-belin/",
-    "twitter": "",
+    "twitter": "https://twitter.com/pierrebelin_",
     "github": "https://github.com/pierrebelin",
     "mastodon": "",
     "sessionize": ""
   },
   "channels": [
     {
-      "name": "Blog",
+      "name": "Site",
       "url": "https://goatreview.com/"
     }
   ],
   "feeds": [
     {
       "type": "LinkTagRssType",
       "url": "https://goatreview.com/rss/"
+    },
+    {
+      "type": "LinkTagRssType",
+      "url": "https://www.youtube.com/feeds/videos.xml?channel_id=UC5LwEdz8usxgvFPiFlelaQA"
```

Comment:
We cannot list it since it is purely in French.

Reply:
I'm only French on my Youtube channel, so I just remove it